### PR TITLE
Add missing operators

### LINKED ISSUE #266: Can't find Expression.cascade method
I saw #22.

### LINKED ISSUE #268: Add "??" BinaryExpression
Happy to make a PR, but I'm not sure what the correct name should be:

```dart
Expression ifNullThen(Expression other) {
  return BinaryExpression._(
    this,
    other,
    '??',
  );
}
```

### PR DESCRIPTION
**Pull request by smkhalsa**
Closes #268
Closes #266

### CODE REVIEW

**Comment by smkhalsa** [comment]
> Thanks! Can you try adding some tests for this? We'll also need to add an entry in the CHANGELOG and bump the version in the pubspec.yaml

@natebosch done

**Comment by smkhalsa** [comment]
@natebosch @kevmoo can we get this merged?

File: pubspec.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-version: 3.2.2-dev
+version: 3.2.3-dev
 
 description: >-
   A fluent, builder-based library for generating valid Dart code
 homepage: https://github.com/dart-lang/code_builder
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: ">=2.6.0 <3.0.0"
```
Comment (kevmoo) [inline pubspec.yaml:9]: please revert this change and the below change

Comment (smkhalsa) [reply pubspec.yaml]: Done. I've also reverted this change in https://github.com/dart-lang/code_builder/pull/277

**Comment by smkhalsa** [comment]
fixes #268 and #266

**Comment by smkhalsa** [comment]
@kevmoo shall I roll this and https://github.com/dart-lang/code_builder/pull/277 into a single PR as 3.3.0-dev?

**Comment by kevmoo** [comment]
> @kevmoo shall I roll this and #277 into a single PR as 3.3.0-dev?

Do that here! Any time you add new features, you should bump the minor version.

**Comment by smkhalsa** [comment]
> Do that here! Any time you add new features, you should bump the minor version.

@kevmoo Ok, I've added those commits, updated the changelog, and bumped the minor version.

**Comment by smkhalsa** [comment]
@kevmoo can we publish 3.3.0-dev to pub.dev please?

**Comment by kevmoo** [comment]
@natebosch wants to do a follow-up. We shouldn't have merged in the `part` stuff w/ this PR

**Comment by natebosch** [comment]
See https://github.com/dart-lang/code_builder/pull/279

**Comment by natebosch** [comment]
We'll also need probably a day to let this go through internal testing.

**Comment by smkhalsa** [comment]
@natebosch any update on getting this published?

**Comment by natebosch** [comment]
I had some test failures to fix up internally and didn't get this synced until yesterday. Things are looking good though so I'll be publishing now.
